youtube_dl/extractor/vgtv.py

```diff
@@ -11,16 +11,19 @@
 
 
 class VGTVIE(InfoExtractor):
-    IE_DESC = 'VGTV and BTTV'
+    IE_DESC = 'VGTV, BTTV, FTV, Aftenposten and Aftonbladet'
     _VALID_URL = r'''(?x)
-                    (?:
-                        vgtv:|
```

Comment:
the newer regex is not compatible with the old use of the shortcut and it's used only by BTArticleIE.